enterprise/maven.jaxws/src/org/netbeans/modules/maven/jaxws/MavenModelUtils.java

```diff
@@ -59,14 +58,28 @@
  */
 public final class MavenModelUtils {
 
+    private static Profile profile;
     private static final String WSIPMORT_GENERATE_PREFIX = "wsimport-generate-"; //NOI18N
     private static final String STALE_FILE_DIRECTORY = "${project.build.directory}/jaxws/stale/"; //NOI18N
     private static final String STALE_FILE_EXTENSION = ".stale"; //NOI18N
-    public static final String JAXWS_GROUP_ID = "org.jvnet.jax-ws-commons"; //NOI18N
+    public static final String JAXWS_GROUP_ID = "com.sun.xml.ws"; //NOI18N
     public static final String JAXWS_ARTIFACT_ID = "jaxws-maven-plugin"; //NOI18N
     public static final String JAXWS_PLUGIN_KEY = JAXWS_GROUP_ID+":"+JAXWS_ARTIFACT_ID; //NOI18N
     private static final String JAXWS_CATALOG = "jax-ws-catalog.xml"; //NOI18N
-    public static final String JAX_WS_PLUGIN_VERSION = "2.3"; //NOI18N
+    public static final String JAXWS_JAKARTAEE_8_PLUGIN_VERSION = "2.3.7"; //NOI18N
+    public static final String JAXWS_JAKARTAEE_9_PLUGIN_VERSION = "3.0.2"; //NOI18N
+    public static final String JAXWS_JAKARTAEE_10_PLUGIN_VERSION = "4.0.3"; //NOI18N
+    
+    public static final String WEBSERVICES_METRO_GROUP_ID = "org.glassfish.metro"; //NOI18N
+    public static final String WEBSERVICES_API_ARTIFACT_ID = "webservices-api"; //NOI18N
+    public static final String WEBSERVICES_RT_ARTIFACT_ID = "webservices-rt"; //NOI18N
+    public static final String WEBSERVICES_API_JAKARTAEE_8_VERSION = "2.4.10"; //NOI18N
+    public static final String WEBSERVICES_API_JAKARTAEE_9_VERISON = "3.0.3"; //NOI18N
+    public static final String WEBSERVICES_API_JAKARTAEE_10_VERISON = "4.0.4"; //NOI18N
+    
+    public static final String MAVEN_PLUGINS_GROUP_ID = "org.apache.maven.plugins"; //NOI18N
+    public static final String WAR_PLUGIN_ARTIFACT_ID = "maven-war-plugin"; //NOI18N
+    public static final String WAR_PLUGIN_VERSION = "2.3.4"; //NOI18N
```

Comment:
I would bump to `3.4.0` if there is not a very good reason to stay on the old version. I remember that I bumped several projects to a 3.X version as 2.X failed when used with JDK 21+